src/screens/FeedScreen/hooks/useCoreFeed.js

```diff
@@ -5,13 +5,12 @@ import {useSafeAreaInsets} from 'react-native-safe-area-context';
 
 import StorageUtils from '../../../utils/storage';
 import useFeedPreloadHook from './useFeedPreloadHook';
+import useMappingPostColorHook from './useMappingColorHook';
 import useViewPostTimeHook from './useViewPostTimeHook';
 import {Context} from '../../../context';
 import {FEEDS_CACHE} from '../../../utils/cache/constant';
-import {checkIsHasColor, hexToRgb} from '../../../utils/colors';
 import {downVote, upVote} from '../../../service/vote';
 import {getFeedDetail, getMainFeedV2WithTargetFeed} from '../../../service/post';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `useMappingPostColorHook` is a good addition as it helps to separate the color mapping logic into its own hook, improving modularity and maintainability. However, please ensure that all necessary dependencies and functions are correctly exported from this new hook and used properly in this file.
```diff
- import {checkIsHasColor, hexToRgb} from '../../../utils/colors';
- import {listFeedColor} from '../../../configs/FeedColor';
+ import useMappingPostColorHook from './useMappingColorHook';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->